spec/fixtures/transactions/show-200.xml

```diff
@@ -2,7 +2,7 @@ HTTP/1.1 200 OK
 Content-Type: application/xml; charset=utf-8
 
 <?xml version="1.0" encoding="UTF-8"?>
-<transaction href="https://api.recurly.com/v2/transactions/abcdef1234567890">
+<transaction href="https://api.recurly.com/v2/transactions/abcdef1234567890" type="credit_card">
```

Comment:
Nit: `type="credit"` is what would come from the API. Doesn't really make a difference for the spec though

Reply:
I added it by checking https://dev.recurly.com/docs/transaction-object
which has `credit_card` type, right?

Reply:
Oh, I was basing it off of this comment
https://github.com/recurly/recurly-client-ruby/blob/master/lib/recurly/transaction.rb#L82

I'm not sure whether the docs are wrong or this comment is wrong, but I would tend to think the docs are probably right... I don't know if I care enough to look into it right this second, leave it like the docs and I'll update that comment in the code eventually if I find out an answer to that